src/vm/jitinterface.h

```diff
@@ -973,6 +973,12 @@ class CEEInfo : public ICorJitInfo
     void* getTailCallCopyArgsThunk(CORINFO_SIG_INFO       *pSig,
                                    CorInfoHelperTailCallSpecialHandling flags);
 
+    bool getTailCallHelp(
```

Comment:
There is no method on JIT/EE interface that is called `getXXXHelp`. The typical naming convention for methods like this is `getXXXInfo`. I would call this `getTailCallInfo`.

Reply:
`getTailCallInfo` sounds like this method will return passive information, but this call is actively asking the VM to create the stubs needed for the tailcall transformation for a specific target function and callsite signature. Also there are many variants of `getXHelper` on this interface, but this call returns multiple helpers and some flags too which is why I did not use that or `getTailCallHelpers`, although maybe this latter one would be ok.

Reply:
I like `getTallCallHelpers`

Reply:
What about for the data structure? `CORINFO_TAILCALL_HELPER_INFO`?

Reply:
If the method is called `getTailCallHelpers`, I would call the structure `CORINFO_TAILCALL_HELPERS`.